### DISCUSSION TITLE
How to increase dpi using single-file-cli?

### DISCUSSION THREAD

**Discussion by buran-energia** [gildas-lormeau/SingleFile#1194] [Q&A]
First of all, say, there is a html page with both normal and high dpi versions of an image with the code like this:
`src="imgs/pic.png" srcset="imgs/pic.png 1x, imgs/pic@2.png 2x"`

is it possible to save both versions?

If not possible, what command would I use to force it to use x2 scaling / dpi?
I downloaded the Z cli version. 
`docker run singlefilez "https://www.wikipedia.org" > wikipedia.html"`
Works fine.
But passing browsers args here gives parsing errors.

**Comment by gildas-lormeau** [comment]
You should be able to keep all the images by setting `--remove-alternative-images=false`

**Comment by buran-energia** [comment]
> You should be able to keep all the images by setting `--remove-alternative-images=false`

It's working. Thank you!

**Comment by gildas-lormeau** [comment]
You're welcome :)
